Automattic/woocommerce-payments
# Revert bumping minimum required version of WC 8.0.0 and WP 6.1

#### Changes proposed in this Pull Request

This PR is to revert WC and WP minimum required version bump done in 25a87d1faf16d11e0c40d9ac48f3f47263efc469 because it is causing problem.

More context can be found in: p1693997746543009-slack-C05RC4FKUAV

#### Testing instructions

I'm not sure how to test this other than making sure GH checks are green and plugin passes smoke testing.
The fixes to the found errors have been included in https://github.com/Automattic/woocommerce-payments/pull/7127, which is the base branch of this PR, so can't verify that the changes from this PR fixes those errors.

Also, make sure the changes from this PR reverts changes from https://github.com/Automattic/woocommerce-payments/pull/7050 except the changelog entry.

-------------------


**Post merge**
